multimodal-rag-assemblyai/main.py

```
@@ -0,0 +1,579 @@
+            duration = int(duration) if duration.isdigit() else 10
+            
+            audio_file = record_audio(duration)
+            if audio_file:
+                print("\n🤔 Processing...")
+                response = process_query("", audio_file)
+                print(f"\n🤖 Response:\n{response}")
+                
+                try:
+                    os.unlink(audio_file)
+                except:
+                    pass
```

Ruff (0.12.2)

Use `contextlib.suppress(BaseException)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(BaseException)`

(SIM105)